sortingQuadraticComplexity/sortingQuadraticComplexity/main.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+
+namespace Sort
```

Comment:
В современном C# лучше писать 
```suggestion
namespace Sort;
```
и дальше без фигурных скобок и лишнего отступа. И лучше объявление неймспейса первой строчкой файла.